runtime/vm/LayoutFFITypeHelpers.cpp

```diff
@@ -203,7 +203,12 @@ LayoutFFITypeHelpers::getStructFFITypeElements(char **layout)
 void
 LayoutFFITypeHelpers::freeStructFFIType(ffi_type *ffiType)
 {
-	if ((NULL != ffiType) && (FFI_TYPE_STRUCT == ffiType->type)) {
+	if ((NULL != ffiType) && (FFI_TYPE_STRUCT == ffiType->type
+#if defined(J9ZOS390)
+			|| FFI_TYPE_STRUCT_FF == ffiType->type
```

Comment:
Please add brackets for the new `if` conditions.
e.g.
```
|| (FFI_TYPE_STRUCT_FF == ffiType->type)
|| (FFI_TYPE_STRUCT_DD == ffiType->type)
```

Reply:
Changed in https://github.com/eclipse-openj9/openj9/pull/19645/commits/1e752555416853814f4beb0cf0eea5f3fe8580bf